draft-ietf-tls-tls13.md

```diff
@@ -2528,8 +2545,8 @@ Legacy algorithms
   in this context with either with RSA using RSASSA-PKCS1-v1_5 or ECDSA.  These values
   refer solely to signatures which appear in certificates (see
   {{server-certificate-selection}}) and are not defined for use in
-  signed TLS handshake messages, even if they appear in the "signature_algorithms"
-  list (this is necessary for backward compatibility with TLS 1.2).
+  signed TLS handshake messages, although they MAY appear in "signature_algorithms"
+  and "signature_algorithms_cert" for backward compatibility with TLS 1.2,
```

Comment:
This is just normalizing to match the language used for PKCS#1 and is not intended to be a behavior change